protobuf/internal/proto_compile.bzl

```diff
@@ -577,17 +719,19 @@ def _proto_compile_impl(ctx):
       _build_grpc_invocation(run, builder)
       _build_grpc_out(run, builder)
 
+  _add_well_known_proto_imports(data, builder)
 
   # Build final immutable compilation unit for rule and transitive beyond
   unit = struct(
     compiler = ctx.executable.protoc,
     data = data,
     transitive_mappings = builder.get("transitive_mappings", {}),
     args = set(builder["args"] + ctx.attr.args),
-    imports = set(builder["imports"]),
+    imports = set(builder["imports"] + ["."]),
```

Comment:
I didn't notice the `root` attr earlier, maybe the default import should be the `root` attr that defaults to "." ?